src/main/kotlin/no/nav/syfo/application/ApplicationEnvironment.kt

```diff
@@ -4,6 +4,7 @@ import io.ktor.application.*
 
 data class Environment(
     val aadDiscoveryUrl: String = getEnvVar("AADDISCOVERY_URL"),
+    val electorPath: String = getEnvVar("ELECTOR_PATH"),
```

Comment:
Er det noe magi et sted som sørger for at ELECTOR_PATH har en verdi?

Reply:
`leaderElection: true` skrur på leader election for en app og da blir den tilgjengelig.